content/wiki/overview-ecosystem-names.md

```diff
@@ -1,65 +1,126 @@
 
-### Metamodern
+The term “Renaissance”, meaning “rebirth”, is a reference to the 'first' Renaissance: the historical period and cultural movement of the 15th and 16th centuries in Europe, which marked the transition from the Middle Ages to early modernity. Like the First Renaissance, we see the Second Renaissance as a period of cultural and societal rebirth of the Western world, characterised by a profound shift at the civilisational level in our most foundational views and values, such as:
+- how we see ourselves and our nature as human beings
+- how we approach the relationship between ourselves and the world around us
+- what we think is meaningful and important 
+
+We have chosen the term "Second Renaissance" for two main reasons. First, it highlights the process of becoming, rather than predefining a destination, which serves as a reminder that the "end-state" is open-ended and emergent, rather than something that can be fixed ahead of time. In doing so, it also overcomes potential conceptual divisions over the nature of the new paradigm that is or ought to emerge. Second, as with the Renaissance of the 15th and 16th centuries, at the core of the Second Renaissance is the relationship between science and spirituality, reason and intuition. The First Renaissance saw a separation of these fundamental human endeavours and ways of knowing; the Second Renaissance seeks to bring about their re-integration.
+
+# Overview of related names & terms
+
+## Metamodern
 
 > “Metamodernism” is a term used to articulate developments in contemporary culture and society that, some argue, represent a move beyond postmodernism. Metamodernism can be characterised as a move from the irony and cynicism characteristic of postmodernism, towards a yearning for sincere and constructive expression and progression. It is the emergence of an ostensible collective desire for change.” [2](https://lifeitself.org/blog/2022/03/11/mapping-metamodern-what-is-metamodernism)
 
-The term was originally coined in 1975 by Mas’ud Zavarzade to describe an emerging cultural trend in American literature. [3](https://nesslabs.com/metamodernism) The term has since been widely used beyond the realm of art and literature, and has become a popular term for referring to the ecosystem. Key thinkers operating under this term include Hanzi Freinacht, Timotheus Vermeulen, Robin van den Akker, Lene Rachel Andersen, and Jason Josephson Storm.
+The term was originally coined in 1975 by Mas’ud Zavarzade to describe an emerging cultural trend in American literature. [3](https://nesslabs.com/metamodernism) Since the publication of Hanzi Freinacht's books *The Listening Society* and *Nordic Ideology* (parts I and II of *A Metamodern Guide to Politics*) in 2017 and 2019, the term 'metamodern' has become a popular way to refer to the Second Renaissance ecosystem and the emerging cultural paradigm that it is seeking to foster. 
```

Comment:
the term 'metamodern' has become a popular way to refer to the Second Renaissance ecosystem **(or something very close to it)** and the emerging cultural paradigm that it is seeking to foster.

-->i'd add something like the bit in bold as not sure that all the names here are exact synonyms of Second Renaissance.